src/lambda/vue/samInvokeFrontend.ts

```diff
@@ -168,18 +172,17 @@ export default defineComponent({
             handler(newval: AwsSamDebuggerConfigurationLoose) {
                 vscode.setState(
                     Object.assign(vscode.getState() ?? {}, {
-                        launchConfig: newval,
+                        launchConfig: JSON.parse(JSON.stringify(newval)),
                     } as SamInvokeVueState)
                 )
             },
             deep: true,
         },
         payload: {
             handler(newval: { value: string; errorMsg: string }) {
-                this.resetJsonErrors()
```

Comment:
Whatever this was used for previously is no longer relevant (and causes the payload error message to disappear)